org.zim_wiki.Zim.json

```diff
@@ -0,0 +1,206 @@
+{
+    "app-id": "org.zim_wiki.Zim",
+    "branch": "stable",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "zim",
+    "rename-desktop-file": "zim.desktop",
+    "rename-icon": "zim",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        /* libappindicator access */
+        "--socket=session-bus",
+        "--filesystem=home"
+    ],
+    "cleanup": ["/share/gtk-doc"],
+    "modules": [
+        {
+            "name": "py2cairo",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./waf configure --prefix=/app",
+                "./waf build",
+                "./waf install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://cairographics.org/releases/py2cairo-1.10.0.tar.bz2",
+                    "sha256": "d30439f06c2ec1a39e27464c6c828b6eface3b22ee17b2de05dc409e429a7431"
+                }
+            ]
+        },
+        {
+            "name": "pygobject",
+            "config-opts": [
+                "--disable-introspection"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.gnome.org/pub/GNOME/sources/pygobject/2.28/pygobject-2.28.7.tar.xz",
+                    "sha256": "bb9d25a3442ca7511385a7c01b057492095c263784ef31231ffe589d83a96a5a"
+                }
+            ]
+        },
+        {
+            "name": "pygtk",
```

Comment:
You need to copy the reconfiguration from here in order to make this build on aarch64 https://github.com/flathub/org.gimp.GIMP/blob/1618d23546fd49d52bd8a3704ebdd4f7e323c1c6/org.gimp.GIMP.base-modules.json#L77